src/Compilers/VisualBasic/Portable/Compilation/VisualBasicCompilation.vb

```diff
@@ -943,7 +943,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
 
                 For Each tree As SyntaxTree In trees
                     If tree Is Nothing Then
-                        Throw New ArgumentNullException(String.Format(VBResources.Trees0, i))
```

Comment:
can this resource be removed?  or is still needed for seomthing else?  does teh C# side do teh same thing?

Reply:
C# side doesn't localize:

https://github.com/dotnet/roslyn/blob/09be0d7882bcc74ed3b832733f701a205936cfe4/src/Compilers/CSharp/Portable/Compilation/CSharpCompilation.cs#L903-L906